README.md

```diff
@@ -719,6 +721,28 @@ const urlCallbackHandler = (url: string): void => {
     res.redirect(url);
 }
 ```
+---
+
+## Logging
+
+The SDK supports the following log levels. `DEBUG` is the lowest log level while `OFF` is the highest.
```

Comment:
Since the log levels are based on the priority the message `DEBUG` should be the lowest log level out of these log levels. The reason `OFF` is the highest log level is because it is intended for turning off the logging.